docs/networking/split-horizon_scenarios.md

```diff
@@ -0,0 +1,95 @@
+
+A Percona Everest administrator can create a set of base domains and generate the necessary TLS certificates in advance. This allows all Percona Everest users to use these domains later without the need to define custom domain names each time they create a MongoDB cluster.
+
+Percona Everest provides an interface to create a Split-Horizon DNS configuration that defines the necessary relationships and stores them for future use.
+
+The following applies to Split-Horizon DNS configs:
+
+- Each config has a unique name within the selected namespace.
+
+- It can only be applied to the Percona Operator for MongoDB Engine. The other database engines do not support this feature.
+
+- Each config is associated with a specific namespace, which the user defines during creation.
+
+**Split-Horizon DNS config parameters**
+
+| Configuration Parameter      | Description                                                                                                         |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| **Name**                      | Each **Split-Horizon DNS config** has a unique name within the selected namespace.                                   |
+| **Namespace**                 | Kubernetes namespace to which this config is applicable. Defined by the user during config creation.                 |
+| **Base domain name**          | Domain name suffix used for generating full domain names for each Pod in the Replica Set. This field is also used as the **Split-Horizon DNS config** name. |
+| **TLS certificates**          | TLS certificate files for the provided base domain. See [instructions](https://docs.percona.com/percona-operator-for-mongodb/tls-manual.html#__tabbed_1_1). <br>- `cert.pem` - certificate<br>- `cert-key.pem` - certificate key<br>- `ca.pem` - certificate authority file |
+
+Once all values are entered, the Percona Everest administrator saves the configuration in the system.
+
+
+## Protect Split-Horizon DNS config with RBAC
+
+A Percona Everest administrator may want to restrict access to the Split-Horizon DNS configuration or the ability to apply a specific Split-Horizon DNS configuration for certain Percona Everest users. To achieve this, the administrator utilizes Percona Everest RBAC to define policies that restrict access to the selected instances. These RBAC policies apply to the entire Split-Horizon DNS configuration.
+
+### Limit access to Split-Horizon DNS config
+
+A Percona Everest administrator may want to allow specific users (for example, Alice) to fully manage (create, edit, read, delete) Split-Horizon DNS configurations.
+Other users can only read the Split-Horizon DNS configs and use them in database clusters, but cannot make any changes.
+
+To implement this, the Percona Everest administrator creates an RBAC policy:
+
+```sh
+p, alice, engine-features/split-horizon-dns-config, *, */*
```

Comment:
please check carefully with the spec: https://www.notion.so/percona/Split-Horizon-DNS-feature-for-PSMDB-27f674d091f3806b9d46dffaa7e24866?source=copy_link#280674d091f3808a8df9d43c815f9850

and the rest of RBAC policies as well